ZelBack/src/services/syncthingService.js

```diff
@@ -101,6 +101,42 @@ async function getConfigFolders(req, res) {
   return res ? res.json(response) : response;
 }
 
+async function getConfigDevices(req, res) {
+  const response = await performRequest('get', '/rest/config/devices');
+  return res ? res.json(response) : response;
+}
+
+async function putConfigFolders(req, res, id, data) {
```

Comment:
The id and data shall be part of req params/query

Reply:
Eg from other fluxos code
```
  let { id } = req.params;
  id = id || req.query.id;
  ```
  then 
  if (!id) {
  // shall be handled if errorMessage of no id specified etc
  }

Reply:
oh, ok, fixing those